providers/oracle/oracleProvider.go

```diff
@@ -141,12 +141,12 @@ func (o *oracleProvider) GetNameservers(domain string) ([]*models.Nameserver, er
 		return nil, err
 	}
 
-	nss := make([]string, len(getResp.Zone.Nameservers))
+	nss := make([]*models.Nameserver, len(getResp.Zone.Nameservers))
 	for i, ns := range getResp.Zone.Nameservers {
-		nss[i] = *ns.Hostname
+		nss[i] = &models.Nameserver{Name: *ns.Hostname}
```

Comment:
I'm requesting that all providers either use `ToNameservers()` or `ToNameserversStripTD()` so that when we change how they are stored, we can just change those functions, not each provider.

If neither of those functions suffice, please let me know.